### DISCUSSION TITLE
Show token for debugging

### DISCUSSION THREAD

**Discussion by static-moonlight** [OpenIDC/mod_auth_openidc#1128] [Q&A] [answered]
Is it somehow possible to take look at an `openid-connect` token, to verify that claims are actually included by the identity server? If yes, how can I do that?

**Comment by zandbelt** [comment, marked as answer]
you can check the logs server side when `LogLevel` is set to include `auth_openidc:debug`, or - to check it client side- you can enable the session info hook with `OIDCInfoHook`, see: https://github.com/OpenIDC/mod_auth_openidc/wiki/Single-Page-Applications#session-info

**Reply by static-moonlight** [reply]
Thanks, the log level was all I needed! I found the information I was looking for in the (default) Apache error log (`/var/log/httpd/error_log`).
